generator/src/test/java/com/scottlogic/deg/generator/helpers/StringGeneratorHelper.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019 Scott Logic Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.scottlogic.deg.generator.helpers;
+
+import com.scottlogic.deg.generator.generation.string.StringGenerator;
+import org.junit.Assert;
+
+import java.util.Iterator;
+
+import static org.hamcrest.Matchers.*;
+
+public enum StringGeneratorHelper {
+    ;
+
+    StringGeneratorHelper() {
+    }
+
+    public static void assertGeneratorCannotGenerateAnyStrings(StringGenerator generator) {
+        Iterator<String> stringValueIterator = generator.generateAllValues().iterator();
+        Assert.assertThat(stringValueIterator.hasNext(), is(false));
+    }
+
+    public static void assertGeneratorCanGenerateAtLeastOneString(StringGenerator generator) {
+        Iterator<String> stringValueIterator = generator.generateAllValues().iterator();
+        Assert.assertThat(stringValueIterator.hasNext(), is(true));
+    }
+
+    public static void assertGeneratorCanGenerateAtLeastOneStringWithinLengthBounds(StringGenerator generator, int minLength, int maxLength) {
```

Comment:
this is the only method in this class that is actually used, and it's only used in one place.

i'd say it would make more sense to put this one method in the RegexStringGeneratorTests class, and delete the rest of this file.